gtsfm/densify/mvsnets/methods/PatchmatchNet/datasets/gtsfm.py

```diff
@@ -0,0 +1,131 @@
+"""Customized MVSDataset class for gtsfm
+    reference: https://github.com/FangjinhuaWang/PatchmatchNet
+
+Authors: Ren Liu
+"""
+
+from torch.utils.data import Dataset
+import numpy as np
+import os
+from PIL import Image
+from .data_io import *
```

Comment:
wildcard imports can lead to unintended consequences, prefer to specify each import explicitly